examples/shell/shell_common/cmd_ping.cpp

```diff
@@ -282,16 +266,19 @@ void StartPinging(streamer_t * stream, char * destination)
     adminInfo = admins.AssignAdminId(gAdminId, kTestControllerNodeId);
     VerifyOrExit(adminInfo != nullptr, err = CHIP_ERROR_NO_MEMORY);
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
     err = gTCPManager.Init(Transport::TcpListenParameters(&DeviceLayer::InetLayer)
-                               .SetAddressType(Inet::kIPAddressType_IPv4)
+                               .SetAddressType(type)
```

Comment:
nit: Use `gDestAddr.Type()` instead of assuming it's IPv4 if IPv4 is enabled?

Reply:
Done